modules/ROOT/pages/configuring-dev-env.adoc

```diff
@@ -53,3 +54,191 @@ include::{partialsdir}/attributes.adoc[]
 == Adding Adding AeroGear SDK modules
 
 
+[role="primary"]
+.Android
+
+****
+
+*Prerequisites*
+
+You will need the following installed on your system
+
+ * Android SDK or Android Studio from https://developer.android.com/studio/index.html[Google]
+
+It is recommended you have the following installed
+
+ * The latest version of http://openjdk.java.net/install/index.html[OpenJDK]
+
+*Using a Published SDK*
+
+_jcenter()_
+
+To use a published version of the SDK you have to add jcenter to the build.grade file in your project's root directory
+
+[source, groovy]
+----
+allprojects {
+    repositories {
+        jcenter() // <-- Add This line
+        google()
+    }
+}
+----
+
+*Reference the Dependencies*
+
+In the build.gradle file of your app you can begin adding dependencies provided by the SDK.
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation 'org.aerogear:android-push:[version]'
```

Comment:
The way we have done it on IOS docs was that we pointed people to individual SDK's.

Reply:
I removed that because this document is included in all the service chapters and i didn't like linking to metrics in e.g. push. But i can add those links back if you think it makes sense.

Reply:
Sorry - yes. This became obvious once I build docs.

Reply:
Can we parameterize [version] here @finp ?

Reply:
Done.